libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -326,6 +327,14 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> i
         'Logs information about the components of the build cache ids for individual operations. This is useful for debugging the incremental build logic.'
     });
 
+    this._showExistingFailureLogsParameter = this.defineFlagParameter({
```

Comment:
We probably shouldn't define this for commands that have `alwaysWatch: true` to avoid end-user confusion.